scipy/stats/tests/test_morestats.py

```diff
@@ -301,6 +304,20 @@ def test_gumbel_r(self):
         assert_array_less(A1, crit1[-2:])
         assert_(A2 > crit2[-1])
 
+    @pytest.mark.parametrize('distname',
+                             ['norm', 'expon', 'gumbel_l', 'extreme1',
+                              'gumbel', 'gumbel_r', 'logistic'])
```

Comment:
If this is just checking success of the fit, could we not combine this with another test that would already loop on every dists?

Reply:
I'm not sure if there is one, but if there is, I'll do that when I fix the mypy issue.

Reply:
There is not one. This is the only one that loops over the distributions.

Reply:
ok let's keep this then.